examples/eks/eks_cluster_gitops/outputs.tf

```diff
@@ -3,6 +3,12 @@ output "cluster_id" {
   description = "CAST AI cluster ID"
 }
 
+output "cluster_token" {
```

Comment:
API Key is very sensitive, why do you suggest to expose it here? It is provided as an input, hence it's available anyway ;)

Reply:
Yes, have passed a flag as sensitive = true. This will not display the api token.

Why do we need this ?

we need to run terraform output cluster_token to get the output and pass to castware helmcharts